### DISCUSSION TITLE
Use Boolean "component" in the blade

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#3963] [Help] [answered]
Is there any option like BooleanColumn that I can use in the blade?

![IBx0Uci](https://user-images.githubusercontent.com/29732308/189549951-537771a3-fad0-4c83-916a-8d136573d997.png)

**Comment by danharrin** [comment, marked as answer]
Yes, we just use `<x-heroicon-o-check-circle>` and `<x-heroicon-o-x-circle>`
